### DISCUSSION TITLE
Welcome to rinohtype Discussions!

### DISCUSSION THREAD

**Discussion by brechtm** [brechtm/rinohtype#335] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jcjohnston** [comment]
Hello, my name is Jason and I live in Tasmania, Australia. I am writing Sphinx-based documentation for a long-term personal project (which looks good in HTML) and I am excited about the possibility of bypassing the whole bloated and archaic TeX/LaTeX ecosystem to create a PDF from it. I have some questions to ask but I'll do that in a separate thread. Good work on rinoh -- it's desperately needed!
